### ISSUE TITLE
Bizhawk runs GBC game as GB game, cannot play GBC movie

### ISSUE THREAD

**Issue by Mand3lbrot** [TASVideos/BizHawk#2149]
Hey, so, the problem I have is very simple. When I load a game into Bizhawk (in this case Pokemon yellow), it says in the top left corner, **GB** - Pokemon Yellow Version, even though it's a gameboy color game. The game itself runs fine, so if I just wanted to play the game casually, this wouldn't be an issue.

However, I when attempting to run a TAS movie for the same game in Bizhawk, it spits out THIS error at me:

_Movie system ID (GBC) does not match the currently loaded platform (GB) unable to load_

I have no Idea what to do to make this error go away. The only thing I could think to do is to delete the GB bios and replace it with a GBC bios, but this still happens. Can you help me out here?

**Comment by YoshiRulz** [comment]
I was able to play back [this Yellow TAS](http://tasvideos.org/3901M.html) on 2.4.2. I tried a few incorrect things but was unable to get the same error. Rename or delete `config.ini` and try again.

**Comment by Mand3lbrot** [comment]
Hey, so I tried using the 2.4.2 BizHawk. Now when I try to load the game it just doesnt work now! Yes, I have the GBC bios, but it seems it wants the GB bios, even though it's a GBC game...
![cool](https://user-images.githubusercontent.com/67115908/85091118-bb21b700-b1b4-11ea-825b-31ceabd323fa.png)
ᴄᴏᴏʟ...

**Comment by RetroEdit** [comment]
Is `libgambatte.dll` in the `dll` folder where you installed BizHawk? That should be there, but maybe it got deleted somehow?

**Comment by zeromus** [comment]
run the prerequisite installer

**Comment by nattthebear** [comment]
@zeromus Should we be probing whatever dll it's missing at startup, or was that one of the 2.5.dev changes we already fixed?

**Comment by YoshiRulz** [comment]
The library is [right here](https://github.com/TASVideos/BizHawk/blob/master/output/dll/libgambatte.dll). Either it's present and not loading due to missing prereqs, or it's been deleted or blocked from loading by anti-virus.

**Comment by nattthebear** [comment]
> 
> 
> The library is [right here](https://github.com/TASVideos/BizHawk/blob/master/output/dll/libgambatte.dll). Either it's present and not loading due to missing prereqs (but all the prereqs are checked by EmuHawk at startup?), or it's been deleted or blocked from loading by anti-virus.

As indicated before, 99.99% chance it's a prereq issue.  The same error message of file not found is shown whether the dependency is not found, or a dependency of the dependency is not found

**Comment by hegyak** [comment]
Steps I did in attempting to recreate this bug/issue:

Load Pokemon Yellow in GBHawk.
Set Sync Settings for Console Mode to "GB"
Reboot Core to apply change.
Record Short Movie file.
Close ROM and End Movie Recording.
Open Pokemon Yellow in GBHawk.
Change Sync Settings for Console Mode to "GBC"
Reboot Core to apply change.
Open Movie that was recorded in "GB" Mode.
Console Mode is forcefully set to "GB" Mode and movie plays.

Repeat above, swapping GB for GBC and Vice Versa.  Movie still plays properly in the original Console Mode that the movie was recorded in.

I used Build:
https://github.com/TASVideos/BizHawk/commit/7f28bc2caffee1093d20bdfd363007e42822dc66
Which was the latest commit as of testing

All ROM/Firmware Files were verified valid and passed their SHA-1 Tests.
